.github/actions/todb-tests/entrypoint.sh

```diff
@@ -51,6 +51,23 @@ for file in "$(ls)"; do
 	fi
 done
 
+# Files added must have date and name later than all existing file
+LATEST_FILE="$(ls -t1 |  head -n 1)"
+LATEST_FILE_TIME="$(stat -f "%m%t%Sm %N" $LATEST_FILE | cut -f1)"
+
+# Get modified times in an array
+mtime_array=()
+arr=("$(ls)")
```

Comment:
Since this is quoted, the entire output of `ls` will be considered a single array element. So, it should be unquoted.

Reply:
ok